### PR TITLE
feature/ads-455 Add useId to svg paths

### PR DESCRIPTION
**Pull request by cblva**
Ref: https://bsc-oit.atlassian.net/browse/ADS-455
Also solves this: https://bsc-oit.atlassian.net/browse/ADS-454

### CODE REVIEW

**Review by APJohns — DISMISSED** [review]

File: src/library/components/icon/icons/IconChevronUp.tsx [hunk 1]
```diff
@@ -11,10 +11,7 @@ const IconChevronUp = (props: IconProps): JSX.Element => {
       xmlns="http://www.w3.org/2000/svg"
       viewBox="0 0 18 18"
     >
-      <path
-        d="M9.7,3.83l8,8a1,1,0,0,1,0,1.4l-.93.93a1,1,0,0,1-1.4,0L9,7.83,2.62,14.17a1,1,0,0,1-1.4,0l-.93-.93a1,1,0,0,1,0-1.4l8-8a1,1,0,0,1,1.4,0Z"
-        fill="currentColor"
```
Comment (kaseyllabelle) [inline src/library/components/icon/icons/IconChevronUp.tsx]: did fill get cut on purpose?

File: src/library/components/icon/icons/IconCardDemo.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { IconProps } from './iconTypes';
 const IconCardDemo = (props: IconProps): JSX.Element => {
   return (
     <svg
-      aria-hidden="true"
+      aria-label="iconCardDemo"
```
Comment (kaseyllabelle) [inline src/library/components/icon/icons/IconCardDemo.tsx:6]: is this intentional?

File: src/library/components/icon/icons/IconCardDemo.tsx [hunk 3]
```diff
@@ -13,10 +13,8 @@ const IconCardDemo = (props: IconProps): JSX.Element => {
       viewBox="0 0 512 384"
     >
       <path
-        d="M464,0H48C21.5,0,0,21.5,0,48v288c0,26.5,21.5,48,48,48h416c26.5,0,48-21.5,48-48V48C512,21.5,490.5,0,464,0z M458,336H54
- c-3.3,0-6-2.7-6-6V54c0-3.3,2.7-6,6-6h404c3.3,0,6,2.7,6,6v276C464,333.3,461.3,336,458,336z M128,88c-22.1,0-40,17.9-40,40
- s17.9,40,40,40s40-17.9,40-40S150.1,88,128,88z M96,288h320v-80l-87.5-87.5c-4.7-4.7-12.3-4.7-17,0L192,240l-39.5-39.5
- c-4.7-4.7-12.3-4.7-17,0L96,240V288z"
+        d="M464,0H48C21.5,0,0,21.5,0,48v288c0,26.5,21.5,48,48,48h416c26.5,0,48-21.5,48-48V48C512,21.5,490.5,0,464,0z M458,336H54c-3.3,0-6-2.7-6-6V54c0-3.3,2.7-6,6-6h404c3.3,0,6,2.7,6,6v276C464,333.3,461.3,336,458,336z M128,88c-22.1,0-40,17.9-40,40s17.9,40,40,40s40-17.9,40-40S150.1,88,128,88z M96,288h320v-80l-87.5-87.5c-4.7-4.7-12.3-4.7-17,0L192,240l-39.5-39.5c-4.7-4.7-12.3-4.7-17,0L96,240V288z"
+        fill="currentColor"
```
Comment (kaseyllabelle) [inline src/library/components/icon/icons/IconCardDemo.tsx:17]: intentional addition of fill?

File: src/library/components/icon/icons/IconMenu.tsx [hunk 4]
```diff
@@ -11,7 +11,10 @@ const IconMenu = (props: IconProps): JSX.Element => {
       xmlns="http://www.w3.org/2000/svg"
       viewBox="0 0 18 18"
     >
-      <path d="M.64,4H17.36A.64.64,0,0,0,18,3.38V1.77a.65.65,0,0,0-.64-.65H.64A.65.65,0,0,0,0,1.77V3.38A.64.64,0,0,0,.64,4Zm0,6.43H17.36A.65.65,0,0,0,18,9.8V8.2a.65.65,0,0,0-.64-.65H.64A.65.65,0,0,0,0,8.2V9.8A.65.65,0,0,0,.64,10.45Zm0,6.43H17.36a.65.65,0,0,0,.64-.65V14.62a.64.64,0,0,0-.64-.64H.64a.64.64,0,0,0-.64.64v1.61A.65.65,0,0,0,.64,16.88Z" />
+      <path
+        d="M.64,4H17.36A.64.64,0,0,0,18,3.38V1.77a.65.65,0,0,0-.64-.65H.64A.65.65,0,0,0,0,1.77V3.38A.64.64,0,0,0,.64,4Zm0,6.43H17.36A.65.65,0,0,0,18,9.8V8.2a.65.65,0,0,0-.64-.65H.64A.65.65,0,0,0,0,8.2V9.8A.65.65,0,0,0,.64,10.45Zm0,6.43H17.36a.65.65,0,0,0,.64-.65V14.62a.64.64,0,0,0-.64-.64H.64a.64.64,0,0,0-.64.64v1.61A.65.65,0,0,0,.64,16.88Z"
+        fill="currentColor"
```
Comment (kaseyllabelle) [inline src/library/components/icon/icons/IconMenu.tsx:16]: intentional addition of fill?

**Pull request closed without merging**